nevergrad/optimization/optimizerlib.py

```diff
@@ -1100,19 +1100,21 @@ class MetaModel(base.Optimizer):
     """Adding a metamodel into CMA."""
 
     def __init__(self, parametrization: IntOrParameter, budget: tp.Optional[int] = None, num_workers: int = 1,
-                 multivariate_optimizer: base.ConfiguredOptimizer = CMA) -> None:
+                 multivariate_optimizer: tp.Optional[base.ConfiguredOptimizer] = None) -> None:
```

Comment:
this should be base.OptCls actually, which works for configured optimizer or not
```suggestion
                 multivariate_optimizer: tp.Optional[base.OptCls] = None) -> None:
```

Reply:
Done.